src/components/behandlerdialog/paminnelse/PaminnelseMelding.tsx

```diff
@@ -11,15 +11,88 @@ import { usePersonoppgaverQuery } from "@/data/personoppgave/personoppgaveQueryH
 import { getAllUbehandledePersonOppgaver } from "@/utils/personOppgaveUtils";
 import { PersonOppgaveType } from "@/data/personoppgave/types/PersonOppgave";
 import { useMeldingTilBehandlerDocument } from "@/hooks/behandlerdialog/document/useMeldingTilBehandlerDocument";
+import { DocumentComponentVisning } from "@/components/DocumentComponentVisning";
+import { ButtonRow, PaddingSize } from "@/components/Layout";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
 
 const texts = {
   button: "Send påminnelse til behandler",
+  send: "Send",
+  cancel: "Avbryt",
 };
 
 const StyledButton = styled(Button)`
   align-self: flex-start;
 `;
 
+const ModalContent = styled(Modal.Content)`
+  padding: 2em;
+`;
+
+const VisOgSendPaminnelse = ({ melding }: PaminnelseMeldingProps) => {
+  const [visPaminnelse, setVisPaminnelse] = useState(false);
```

Comment:
Kanskje `visPaminnelseModal`? Det gis kanskje av contexten for resten av koden her at det er modalen, men jeg tenkte først det var den blå knappen man skulle skjule/vise og da var jeg litt confused 🤔